src/utils/macros.rs

```diff
@@ -7,7 +7,7 @@ macro_rules! use_prelude { () => (
 )}
 
 /// I really don't get the complexity of `cfg_if!`…
-macro_rules! cfg_match {
+macro_rules! cfg_matches {
```

Comment:
Sorry for the nit, but could you go with:

```suggestion
macro_rules! match_cfg {
```

instead? `matches` conveys too much of a boolean meaning to me 😄